ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -1,13 +1,29 @@
 base_images:
-  centos7:
-    name: centos
-    namespace: openshift
-    tag: "7"
+  centos:
+    name: "4.4"
+    namespace: ocp
+    tag: base
```

Comment:
This is the 4.4 base image, right?  Maybe `centos` -> `base` above?

Reply:
Ah, correct. Replaced it with actual centos image from `ci` namespace